### DISCUSSION TITLE
Created and modified Timestamp

### DISCUSSION THREAD

**Discussion by SIXASIS** [kimai/kimai#3577] [General]
Is there any way to add the fields:
- Created Timestamp
- Modified Timestamp
- Modified By

To the tables and for the APIs, i believe thats really easy to add it to the tables (at least modified and created datetime) but the issue is that its not so easy to get it showed at the APIs and i trully believe that this is a really important information to show in the web interface and at the API results, not only for the Timesheets, also for Activities and Projects.

**Comment by kevinpapst** [comment]
Sorry, but I have no plans to add these fields.
